.github/workflows/test.yaml

```diff
@@ -6,7 +6,7 @@ on:
       - main
 jobs:
   main:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
latest is better, no?

Reply:
OTP 21.3.8.24 is not available on ubuntu 22, so I pointed to the one that worked.